src/tz_haiku.rs

```diff
@@ -0,0 +1,23 @@
+#[cxx::bridge(namespace = "tz_haiku")]
+mod ffi {
+    unsafe extern "C++" {
+        include!("iana-time-zone/include/impl_haiku.h");
+
+        /// SAFETY: `buf` must be valid and be at least `buf_len` bytes long
+        unsafe fn get_tz(buf: *mut u8, buf_len: usize) -> usize;
```

Comment:
This function can be implemented much simpler without using `unsafe` or casting slices to pointers manually: https://cxx.rs/binding/slice.html. I did not notice this feature yesterday. Will change it this evening or tomorrow.

Reply:
done in https://github.com/strawlab/iana-time-zone/pull/66/commits/742f223624a46326a7e4d304d4894c6c38950742